app/controller/sdl/Abstract/Controller.js

```diff
@@ -1632,6 +1636,18 @@ SDL.SDLController = Em.Object.extend(
           frames[i].style.pointerEvents = "none";
         }
       }
-    }
+    },
+
+    showWebViewApp: function(appID) {
+      for(var key in SDL.SDLModel.webApplicationFramesMap) {
+        SDL.SDLModel.webApplicationFramesMap[key].hidden = key != SDL.SDLModel.appIDtoPolicyAppIDMapping[appID];
+      }
+    },
+
+    hideWebApps: function() {
```

Comment:
@YarikMamykin please add a description

Reply:
@AKalinich-Luxoft description added in https://github.com/LuxoftSDL/sdl_hmi/pull/39/commits/a725b2ecd972b868047011818fc0e1b26fb2b29a